CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Mon Jan 11 2021 Chris Tessmer <chris.tessmer@onyxpoint.com> - 7.0.2
+- Removed EL6 support
```

Comment:
This component does not support EL6.  This comment is not appropriate.  Looks like only changes to be noted are formatting in the metadata.json.